app/preact/src/client/preview/render.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable-next-line no-unused-vars */
+import { h, render } from 'preact';
```

Comment:
will we have the pragma h pollution everywhere in storybook ?
Just need to make sure